CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 ### Bug fixes
 
 - [PBD] Obfuscate cifs password from device config [#4384](https://github.com/vatesfr/xen-orchestra/issues/4384) (PR [#4401](https://github.com/vatesfr/xen-orchestra/pull/4401))
+- [Backup NG/New schedule] Fix errors displayed behind the form [#3831](https://github.com/vatesfr/xen-orchestra/issues/3831) (PR [#4131](https://github.com/vatesfr/xen-orchestra/pull/4131))
```

Comment:
The errors still show up in the background when you click OK, right?
Can't we do a more generic fix (in another PR) for the form modals maybe? Like finding a way to properly show errors when the modal is still open. We could also disable the OK button with some condition. What do you think?

Reply:
I agree. What do you suggest to do in this PR?

Reply:
I'd say we can leave the implementation like that for this PR and change the Changelog entry with something like "Properly show user errors in the form".